conans/client/cache.py

```diff
@@ -257,6 +257,21 @@ def delete_empty_dirs(self, deleted_refs):
                         break  # not empty
                 ref_path = os.path.dirname(ref_path)
 
+    def remove_package_system_reqs(self, reference):
+        assert isinstance(reference, ConanFileReference)
+        conan_folder = self.conan(reference)
+        system_reqs_folder = os.path.join(conan_folder, SYSTEM_REQS_FOLDER)
+        if not os.path.exists(conan_folder):
+            raise ValueError("%s does not exist" % repr(reference))
+        if not os.path.exists(system_reqs_folder):
+            return
+        if not os.access(system_reqs_folder, os.W_OK):
```

Comment:
Not necessary to check permissions here. Just use `conans.util.files.rmdir` instead of the following `shutil.rmtree`. It tries to change permissions even. Then you can try except Exception and say:
```python
   try:
      rmdir(system_reqs_folder)
   except Exception as e:
        raise ConanException("Unable to remove system requirements at %s" % system_reqs_folder)
```